src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -563,6 +561,19 @@ public function getValidPropertyPaths()
         ];
     }
 
+    public function getValidReadPropertyPaths()
+    {
+        $testCases = $this->getValidWritePropertyPaths();
+
+        // Optional paths can only be read and can't be written to.
```

Comment:
Should we test the writing too?

Reply:
Nullsafe operator is not allowed in in write context. see: https://wiki.php.net/rfc/nullsafe_operator#nullsafe_operator_in_write_context

Not sure how to test that if needed